### PR TITLE
New Features: aliases, banners, dns, VM tracer, sflow, ssh, console, logging, tacacs, aaa, snmp, standard IP ACL, ipv6 routing, ipv6 static routes, multicast, pim sparse mode, rp and anycast rp, router l2 vpn, BGP AF

### PR DESCRIPTION
**Pull request by carlbuchmann**
**Created EOS CLI templates for**
- Terminal Settings
- Aliases
- Banners
- Domain Lookup
- DNS Domain
- VM Tracer Sessions
- Sflow: generic config not per port
- Management Console
- Management SSH
- IPv6 Routing: enabling ipv6 for GRT and VRFs
- IPv6 Static Routes
- Standard IP ACL
- Tacacs+ Servers
- AAA Server Groups
- IP Tacacs+ Source Interface
- SNMP Settings: hosts, groups, users, views, location, contact, traps
- L2 Router VPN
- IGMP Snooping: vlan
- Router Multicast: ipv4
- PIM Sparse Mode
- Anycast RDV Point
- RDV Point

**Refactored**
- BGP to add new address-families

**Added features in**
- Logging: logging server, logging trap, logging buffer
- Management Interface: IPv6
- Management API HTTP: ACL
- SVI: ip helper, pim sparse-mode, IPv6 enable,IPv6, IPv6 link local, IPv6 router virtual, nd ra,spt portfast, spt bpdufilter, shutdown, multicast ipv4 source route export, pim local-interface 
- Static Routes: tags, administrative distance and description
- Queue Monitor Length : notifying
- Route-maps: set and descriptions
- Loopback Interfaces: IPv6, shutdown
- Port-channel Interfaces: IPv6 enable, IPv6, IPv6 link local, nd ra, spt portfast, spt bpdufilter, vmtracer, shutdown, qos trust
- Ethernet Interfaces: IPv6, IPv6 link local, IPv6 enable, IPv6 nd ra, pim, spt portfast, spt bpdufilter, vmtracer, shutdown, qos trust
- MLAG: disable dual-primary detection

**Notes**
eos-intended-config.j2 and README.md have been updated accordingly.
EOS documentation jinja templates are being updated, but it is not completed yet.

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/loopback-interfaces.j2 [hunk 1]
```diff
@@ -9,6 +11,16 @@
 {%-        if loopback_interfaces[loopback_interface].ip_address is defined %} {{ loopback_interfaces[loopback_interface].ip_address }} {% else %} - {% endif -%} |
 {%     endfor %}
 
+IPv6
```
Comment (titom73) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/loopback-interfaces.j2:14]: IPv6 section appears even if no v6 address is configured on loopback. Other IPv6 sections display information to inform IPv6 is not configure. We should apply same behavior everywhere

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-interfaces.j2 [hunk 2]
```diff
@@ -8,8 +11,19 @@
 | {{ management_interfaces[management_interface].ip_address }} | {{ management_interfaces[management_interface].gateway }} |
 {% endfor %}
 
+IPv6
```
Comment (titom73) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-interfaces.j2:14]: IPv6 section appears even if no v6 address is configured on interfaces. Other IPv6 sections display information to inform IPv6 is not configure. We should apply same behavior everywhere

**Comment by titom73** [comment]
Tested with following scenario all from eos_l3ls_evpn role :

- eAPI deployement: __OK__
- CVP deployment: __OK__
- BGP as underlay: __OK__
- ISIS as underlay (Unsupported): __OK__ (no error reported)
- OSPF as underlay: __OK__

**Review by titom73 — APPROVED** [review]
LGTM

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/loopback-interfaces.j2 (hunk 1 above)
Comment (carlbuchmann) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/loopback-interfaces.j2:14]: agreed, will update accordingly!

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-interfaces.j2 (hunk 2 above)
Comment (carlbuchmann) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-interfaces.j2:14]: agreed, will update accordingly!

**Comment by titom73** [comment]
CI is broken due to ansible-test registry downtime

```
Error response from daemon: Get https://quay.io/v2/ansible/default-test-container/manifests/latest: received unexpected HTTP status: 500 Internal Server Error
```

**Review by aphillipps — APPROVED** [review]
Fine, tested with 4.22.5M and 4.23.0F

**Comment by carlbuchmann** [comment]
Tested with AVD example and will merge!

**Pull request merged**